presto-orc/src/test/java/io/prestosql/orc/BenchmarkStreamReaders.java

```diff
@@ -261,6 +342,19 @@ public Object readSliceDictionaryWithNull(VarcharWithNullBenchmarkData data)
         return blocks;
     }
 
+    @Benchmark
+    public Object readSliceDictionaryWithAllNull(VarcharWithAllNullBenchmarkData data)
```

Comment:
You might be able to have a single method all of these delegate to that takes the type variable.